data/plugins/autocomplete.lua

```diff
@@ -619,6 +695,19 @@ function autocomplete.can_complete()
   return false
 end
 
+---Register a font icon that can be assigned to completion items.
+---@param name string
+---@param character string
+---@param font? renderer.font
+---@param color? string One of the color names from style.syntax
```

Comment:
Why not just using a color directly?

Reply:
To keep the colors assigned to the autocomplete icon in sync with live color scheme changes.

Reply:
It could take a string or a color.

Reply:
> It could take a string or a color.

that would be useful when the user is customizing the assigned colors, will implement